UpdateDependencies.ps1

```diff
@@ -10,30 +10,37 @@ param(
     [Parameter(Mandatory=$true)][string]$GitHubUser,
     [Parameter(Mandatory=$true)][string]$GitHubEmail,
     [Parameter(Mandatory=$true)][string]$GitHubPassword,
-    [Parameter(Mandatory=$true)][string]$CoreFxVersionUrl, 
+    [Parameter(Mandatory=$true)][string]$CoreFxVersionUrl,
+    [string[]]$DirPropsVersionElements = 'CoreFxExpectedPrerelease',
     [string]$GitHubUpstreamOwner='dotnet', 
     [string]$GitHubOriginOwner=$GitHubUser,
     [string]$GitHubProject='corefx',
     [string]$GitHubUpstreamBranch='master',
     # a semi-colon delimited list of GitHub users to notify on the PR
     [string]$GitHubPullRequestNotifications='')
 
-$CoreFxLatestVersion = Invoke-WebRequest $CoreFxVersionUrl -UseBasicParsing
-$CoreFxLatestVersion = $CoreFxLatestVersion.ToString().Trim()
+$LatestVersion = Invoke-WebRequest $CoreFxVersionUrl -UseBasicParsing
```

Comment:
Do we always expect every dependency to have a version matching the CoreFx version?